pkgs/development/python-modules/scrapinghub/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage, fetchPypi, lib
+, requests, retrying, six
+, glibcLocales
+}:
+
+buildPythonPackage rec {
+  pname = "scrapinghub";
+  version = "2.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08yhx88ihfhq4snywq9hjy9l6y0cm51cwnb69hc2f0k5qayfpqlh";
+  };
+
+  propagatedBuildInputs = [ requests retrying six ];
+
+  # Prevent unicode decoding error in setup.py
+  buildInputs = [ glibcLocales ];
+  LC_ALL = "en_US.UTF-8";
+
+  # Tests are network-based
+  doCheck = false;
```

Comment:
They don't have tests scraping from localhost?

Reply:
Not sure, but this is not the scraping library. Rather, the library that manages the crawlers running in their cloud environment, like aws-cli, I'd say.
Anyway, I see lots of messages like this in the tests:
```
pool_timeout = None, release_conn = False, chunked = False, body_pos = None
response_kw = {'decode_content': False, 'preload_content': False}, conn = None
release_this_conn = True, err = None, clean_exit = False
timeout_obj = <urllib3.util.timeout.Timeout object at 0x7ffff537ab00>
is_new_proxy_conn = False
```

Reply:
Don't know what that is, probably just test-related.

Reply:
Well, I disabled the tests that give a problem. At least some are passing :)